raiden/network/transport/matrix.py

```diff
@@ -702,6 +734,8 @@ def _handle_presence_change(self, event):
         if address not in self._address_to_userids:
             return
         self._address_to_userids[address].add(user_id)
+        # maybe inviting user used to also possibly invite user's from discovery presence changes
+        self._maybe_invite_user(user)
```

Comment:
Shouldn't we only invite if the state is not offline?

Reply:
Part of this PR is exactly about being able to handle invites sent while node is offline (which is pretty good for reliability of matrix transport).